design/environment.go

```diff
@@ -41,10 +44,10 @@ var envAttrs = a.Type("EnvironmentAttributes", func() {
 	})
 })
 
-var envRelationships = a.Type("EnvironmentRelations", func() {
-	a.Attribute("space", relationGeneric, "Environment associated with one space")
-	// TODO for type
-})
+// var envRelationships = a.Type("EnvironmentRelations", func() {
+// a.Attribute("space", relationGeneric, "Environment associated with one space")
+// TODO for type
+// })
```

Comment:
@alexeykazakov there is no more relationship in goa-design between environment and space .. is this the way we wanted to model it .. right ??

Reply:
Well... If I understand JSON API correctly then spaceID should be inside "relationships". Not "attributes". But on the other hand I think we can skip adding spaceID into the env payload completely for now.
The spaceID is part of the URL anyway: GET/POST `/api/spaces/:spaceID/environments`
We can add spaceID to the payload later if/when needed.

Reply:
removed `spaceID` attr from goa-design [2eab32d](https://github.com/fabric8-services/fabric8-env/pull/2/commits/2eab32d43b41c3bac7e5e63c7a9e647b78e0e822)

Reply:
`spaceId` removed [2eab32d](https://github.com/fabric8-services/fabric8-env/pull/2/commits/2eab32d43b41c3bac7e5e63c7a9e647b78e0e822)